src/i18n/locales/en.ts

```diff
@@ -79,6 +79,9 @@ export class en implements Locale {
   commaX0ThroughX1() {
     return ", %s through %s";
   }
+  commaAndX0ThroughX1() {
```

Comment:
It looks like we'll need to add this to all i18n files.  We can grab the translation of "and" from `the `spaceAnd` item here.